indico/modules/events/papers/templates/management/cfp_actions/ended.html

```diff
@@ -0,0 +1,26 @@
+{% extends 'events/papers/management/cfp_actions/_base.html' %}
+
+{% block label %}
+    {% trans -%}
+        The call for papers is closed
+    {%- endtrans %}
+{% endblock %}
+
+{% block text %}
+    {% set tz = event.timezone %}
+    {% trans date=cfp.end_dt|format_date(timezone=tz), time=cfp.end_dt|format_time(timezone=tz) -%}
+        Closed on <strong>{{ date }}</strong> at <strong>{{ time }}</strong> ({{ tz }})
```

Comment:
`.` missing at the end of sentence. Just to keep consistency with `not_started.html` and `open.html`.